src/components/forms/account/AccountForm.tsx

```diff
@@ -30,6 +30,7 @@ export type AccountFormProps = {
   action?: 'add' | 'update' | 'delete',
   onSave: Function,
   defaultValues?: Partial<FormValues>,
+  hideDefaults?: boolean,
```

Comment:
this new field allows to hide some of the fields of the form when defaults are provided. It helps to simplify forms when we know some values already (i.e. if there's only one currency, no need to show it)